### PR TITLE
Fix outdated applio info

### PR DESCRIPTION
**Pull request by aitronz**
In this second pull request, I have replaced all the subjective elements and reviewed the grammar and vocabulary. For instance, I have made the following changes from the previous pull request(and some more):

"The fastest of all compared Forks" has been revised to "Very fast". (If you believe that "Very fast" isn't objective, I can remove it from the list of advantages.)
"Ultimate voice cloning tool" has been updated to "VITS-based Voice Conversion Tool".

### CODE REVIEW

File: assets/Essentials/Voice Models.md [hunk 1]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
 
 - If it only exists in weights.gg, download the .ZIP & <u>[upload it to HF](https://docs.aihub.wtf/essentials/voice-models/#uploading-to-hugging-face)</u>.  
 
++++ applio.org   
+:::content-center  
+<img src="../searchrvcmodels-img/applio-web-2.png" alt="image" width="350" height="auto">           
+:::
+
+- This a website where people can upload voice models.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md]: Either This **is** a website... or just A website, This a... is not correctz

File: assets/Essentials/Voice Models.md [hunk 2]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
...
+***
+###### ‎
+#### 2. Search
+- Type the name of the model in the ``Search`` bar & click a result.     
+      
+    <img src="../searchrvcmodels-img/applio-web-1.png" alt="image" width="1000" height="auto"> ‎    
+‎   
+
+!!!
+If you get models from different years, remember, the person's voice changes overtime.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md:120]: ??

File: assets/Essentials/Voice Models.md [hunk 3]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
...
+
+    <img src="../searchrvcmodels-img/applio-web-3.png" alt="image" width="1000" height="auto">
+
+!!!success
+If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/rvc/resources/epochs--tensorboard/)</u>.
+!!!
+***
+###### ‎
+#### 4. Download       
+- Click on ``Download model``. It will download a .ZIP file of it.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md]: This will sounds better

File: assets/Essentials/Voice Models.md [hunk 4]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
...
+***
+###### ‎
+#### 3. Evaluate model (optional)      
+- Check the description, likes or tags, this can give you a slight idea of how good the model is.
+- This step is especially useful if you get multiple results from the same model.
+
+    <img src="../searchrvcmodels-img/applio-web-3.png" alt="image" width="1000" height="auto">
+
+!!!success
+If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/rvc/resources/epochs--tensorboard/)</u>.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md]: you can learn more <here> is better

File: assets/Essentials/Voice Models.md [hunk 5]
```diff
@@ -95,6 +96,51 @@ If you're curious about the epochs, <u>[learn more here](https://docs.aihub.wtf/
...
+!!!
+***
+###### ‎
+#### 4. Download       
+- Click on ``Download model``. It will download a .ZIP file of it.     
+‎  
+<img src="../searchrvcmodels-img/applio-web-4.png" alt="image" width="600" height="auto">‎                          
+‎   
+- If you need a link for it, use the other methods.
```
Comment (SleepyYui) [inline assets/Essentials/Voice Models.md:140]: ??

File: assets/RVC/Cloud/Applio Colab.md [hunk 6]
```diff
@@ -29,13 +29,15 @@ order: 5000
...
+- Very fast
 - TTS features            
 - Automatic model upload
 - Has Mangio-Crepe
 - User-friendly UI
+- TensorBoard included
+- Extra features(plugins, model fusion, etc)
 ||| ❌ **CONS** 
-- A little slower compared w/ forks
-- More unstable
```
Comment (SleepyYui) [inline assets/RVC/Cloud/Applio Colab.md:38]: 37,38 have to be proven since I can neither confirm nor deny

File: assets/RVC/Local/Applio.md [hunk 7]
```diff
@@ -30,23 +30,33 @@ order: 5000
...
 - Very complete
 - Has an active development
+- Currently stable
+- Very fast
 - TTS features            
 - Automatic model upload
 - Has Mangio-Crepe
 - User-friendly UI
 - TensorBoard included
+- Extra features(plugins, model fusion, etc)
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md]: space and colon between features and the opening bracket

File: assets/RVC/Local/Applio.md [hunk 8]
```diff
@@ -30,23 +30,33 @@ order: 5000
...
 ||| ❌ **CONS** 
-- A little slower compared w/ forks
-- More unstable
-||| 
+- Issues when downloading it in external drives
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md]: replace it in with to

File: assets/RVC/Local/Applio.md [hunk 9]
```diff
@@ -30,23 +30,33 @@ order: 5000
...
+- Very fast
 - TTS features            
 - Automatic model upload
 - Has Mangio-Crepe
 - User-friendly UI
 - TensorBoard included
+- Extra features(plugins, model fusion, etc)
 ||| ❌ **CONS** 
-- A little slower compared w/ forks
-- More unstable
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md:40]: once again, can't confirm nor deny

File: assets/RVC/Local/Applio.md [hunk 10]
```diff
@@ -30,23 +30,33 @@ order: 5000
...
 ## Download :icon-download:
 :::
 ###### ‎
-1. Easiest way to download it is by going to their Hugging Face repo <u>[here](https://huggingface.co/IAHispano/Applio/blob/main/Compiled/ApplioV3.0.8.zip)</u>, and click [ :icon-download: **download** ] on the right.
+!!!warning Before Downloading:
+- Make sure that you place Applio inside a folder on C drive.
+- Don't put it in a folder with privileged access.
+- Don't run the run-install.bat as an administrator.
+- Make sure the path does not contain any spaces or special characters.
+- Desactivate your antivirus and firewall to avoid missing dependencies.
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md]: Disable or Deactivate, Desactivate doesn't exist

File: assets/RVC/Local/Applio.md [hunk 11]
```diff
@@ -30,23 +30,33 @@ order: 5000
...
+!!!
+***
+1. Easiest way to download it is by going to their Hugging Face repo <u>[here](https://huggingface.co/IAHispano/Applio/tree/main/Compiled)</u>, and click [ :icon-download: **download** ] on the right.
```
Comment (SleepyYui) [inline assets/RVC/Local/Applio.md]: Replace with
```md
The easiest way to download Applio is by going to Applio's <u>[Hugging Face repo](https://huggingface.co/IAHispano/Applio/tree/main/Compiled)</u>, and clicking the [ :icon-download: **download** ] button on the right-hand side.
```

**Review by SleepyYui — CHANGES_REQUESTED** [review]

**Comment by SleepyYui** [comment]
the things I cannot confirm can stay btw, I'll just ask a few people on their opinion regarding that

**Comment by aitronz** [comment]
I'm sorry if there are a bunch of commits, GitHub in mobile is horrendous

**Comment by SleepyYui** [comment]
Will look at it later today, should be fine by what I've seen by now

**Review by SleepyYui — APPROVED** [review]
few gramar issues still but doesn't matter

**Pull request merged**